src/presenters/point-presenter.js

```diff
@@ -86,41 +89,49 @@ export default class PointPresenter {
   #escKeyDownHandler = (evt) => {
     if (evt.key === 'Escape') {
       evt.preventDefault();
-      this.#closeForm();
+      this.#replaceEditFormToPoint();
     }
   };
 
   #replacePointToEditForm() {
     replace(this.#editPointComponent, this.#pointComponent);
     this.#handleModeChange();
     this.#mode = Mode.EDITING;
+    document.addEventListener('keydown', this.#escKeyDownHandler);
   }
 
   #replaceEditFormToPoint() {
     replace(this.#pointComponent, this.#editPointComponent);
     this.#mode = Mode.DEFAULT;
-  }
-
-  #closeForm() {
-    this.#replaceEditFormToPoint();
     document.removeEventListener('keydown', this.#escKeyDownHandler);
   }
 
   #handleEditClick = () => {
     this.#replacePointToEditForm();
-    document.addEventListener('keydown', this.#escKeyDownHandler);
   };
 
   #handleFavoriteClick = () => {
     this.#handleDataChange({...this.#point, isFavorite: !this.#point.isFavorite});
   };
 
   #handleCloseCLick = () => {
-    this.#closeForm();
+    this.#replaceEditFormToPoint();
+  };
+
+  #handleFormSubmit = (update) => {
+    this.#handleDataChange(
+      UserAction.UPDATE_POINT,
+      isPatchUpdate(this.#point, update) ? UpdateType.PATCH : UpdateType.MINOR,
```

Comment:
В функциях вычислять значение аргументов не оч, был бы линтер посильнее закручен ударил бы по рукам) Лучше создать переменную с тем же условием а здесь ее использовать
```suggestion
      updateType,
```